build.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/sh
 set -e -x
 
+: ${DOCKER_REGISTRY:="dispatchframework"}
+
 cd $(dirname $0)
 
-docker build -t dispatchframework/nodejs-base:0.0.8 .
+IMAGE=${DOCKER_REGISTRY}/nodejs-base:$(cat version.txt)
```

Comment:
This makes version references in ./README.md out of sync. Is there a way to include a file in markdown too?